### PR TITLE
state loki entrance

### LINKED ISSUE #95: basic dashboard request
Is there any basic dashboard example?

I really want to try it with a dashboard example.

### PR DESCRIPTION
**Pull request by daixiang0**
New users may think loki need a dashboard like prometheus, but no indeed.
So add a entrance statement about it.

Fix #95 

Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ Grafana ships with built-in support for Loki in the [latest nightly builds](http
 3. Click the `+ Add data source` button in the top header.
 4. Choose Loki from the list.
 5. The http URL field should be the address of your Loki server e.g. `http://localhost:3100` and `http://loki:3100` when running with docker and docker-compose.
+6. Click the `Explore` not `Dashboard`, enter `__filename__="/var/log/docker.log"` then you can get docker log in UI.
```
Comment (davkal) [inline README.md]: Let's rephrase this to stay independent on what labels loki provides. I suggest:

`6. To see the logs, click "Explore" on the sidebar, select the Loki datasource, and then choose a log stream using the "Log labels" button.`

Comment (daixiang0) [reply README.md]: done as you suggest.
